src/consensus/aft/raft.h

```diff
@@ -1891,6 +1979,19 @@ namespace aft
 
       channels->send_authenticated(
         to, ccf::NodeMsgType::consensus_msg, response);
+
+      if (
+        consensus_type == ConsensusType::BFT &&
+        response.success == AppendEntriesResponseType::OK)
+      {
+        NodeCaughtUpMsg ncamsg = {
+          {raft_node_caught_up}, state->current_view, state->last_idx};
```

Comment:
AIUI, this new message only contains the current view and last idx, which are already sent as part of the `AppendEntriesResponse`. Couldn't we use the `AppendEntriesResponse` only for this purpose instead of creating a new message type?